consul/tests/common.py

```diff
@@ -46,8 +45,6 @@
     'consul.memberlist.pushpullnode.',
     'consul.raft.commitTime.',
     'consul.raft.leader.dispatchLog.',
-    'consul.raft.replication.appendEntries.rpc.',
-    'consul.raft.replication.heartbeat.',
```

Comment:
Moved to https://github.com/DataDog/integrations-core/pull/9382/files#diff-0417f5405c70f9a67519cb3926358f6100763016af6f781fa24b8e1043bf29a3R125-R130